cumulusci/tasks/bulkdata/tests/test_extract.py

```diff
@@ -524,19 +524,23 @@ def test_soql_for_mapping(self):
         task = _make_task(
             ExtractData, {"options": {"database_url": "sqlite:///", "mapping": ""}}
         )
-        mapping = {
-            "sf_object": "Contact",
-            "oid_as_pk": True,
-            "fields": {"Id": "sf_id", "Test__c": "Test"},
-        }
+        mapping = MappingStep(
+            **{
```

Comment:
This is a strange way of constructing the keyword arguments. I would prioritize readability and make the switch to passing these as individual keyword args.